ocean_provider/routes/decrypt.py

```diff
@@ -90,9 +90,14 @@ def decrypt():
         description: decrypted document
       400:
         description: One or more of the required attributes are missing or invalid.
+      405:
```

Comment:
please document the new status codes as well, in the readmes.

Reply:
do not forget about the error readme :) https://github.com/oceanprotocol/provider/blob/main/ocean_provider/routes/README.md

Reply:
I have put them there as well :)